src/routes/(blog-article)/+layout.svelte

```diff
@@ -69,7 +69,11 @@
 					{#if post.tags?.length}
 						<div class="tags">
 							{#each post.tags as tag}
-								<Tag>{tag}</Tag>
+								<Tag>
+									<a href="/tags/{tag}">
+										{tag}
+									</a>
+								</Tag>
```

Comment:
Hi @grmbyrn do you think we could make all tags "linkable" and put the link inside the component?

Reply:
Hi @josecelano I moved the link for tags from `+layout.svelte` in `(blog-article)` route to `Tag` component.

This caused a couple of problems with tags in other parts of the site as the component wasn't fetching the appropriate data when the route changed, leaving the screen blank unless it was refreshed. Using `<a data-sveltekit-reload href="/path">Path</a>` as recommended [here](https://kit.svelte.dev/docs/link-options) solved the issue.